packages/7-todo-tdd/src/components/todo-item/TodoItem.test.tsx

```diff
@@ -0,0 +1,88 @@
+import { ITodoDTO } from 'fed-todo-journey_todo-common';
+import { TodoItemDriver } from './TodoItem.test.driver'
+import { Chance } from 'chance';
+import { enzymeContainerSetupAndTeardown } from '../../test/utils';
+
+const chance = new Chance();
+
+describe('todo item content should be rendered properly', () => {
+    let todoItemDriver: TodoItemDriver;
+    let todo: ITodoDTO;
+
+    beforeEach(async () => {
+        todo = { content: chance.word(), _id: chance.guid(), userId: chance.guid(), isDone: false };
```

Comment:
move out from the before into a function 
`buildTodo(todo: Partial<ITodoDTO>)` the params that are not given will get a default chance values